config/locales/client.en.yml

```diff
@@ -1542,8 +1542,12 @@ en:
         remove_banner: "Remove Banner Topic"
 
       reply:
-        title: 'Reply'
-        help: 'begin composing a reply to this topic'
+        message:
+          title: 'Reply'
+          # help i.e. tooltop is at controls.reply
+        thread:
```

Comment:
Hmm. Now that I think of it, I think that "topic" instead of "thread" would follow the naming convention...